datafusion/core/src/execution/mod.rs

```diff
@@ -41,12 +41,14 @@
 //!
 
 pub mod context;
-pub mod disk_manager;
-pub mod memory_pool;
 // backwards compatibility
 pub use crate::datasource::file_format::options;
-pub mod registry;
 pub mod runtime_env;
 
+// backwards compatibility
+pub use datafusion_execution::disk_manager;
```

Comment:
I moved three modules in this first PR -- runtime_env will take a little more work as will `context` so I plan to do them as smaller follow on PRs